words/objects.txt

```diff
@@ -292,6 +387,8 @@ bubble
 bubbler
 buckaroo
 bucket
+buckthorn
+pepperberry
```

Comment:
What happened to the now-canonical `buckthornpepperberry`?

Reply:
fixed!